src/wallet/wallet.cpp

```diff
@@ -2513,26 +2513,21 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const CoinEligibil
 
     // Start with BnB. If that fails, use SRD
     if (SelectCoinsBnB(utxo_pool, nTargetValue, cost_of_change, setCoinsRet, nValueRet, not_input_fees)) {
-        bnb_used = true;
         return true;
     } else {
-        bnb_used = false;
         return SingleRandomDraw(nTargetValue, utxo_pool, setCoinsRet, nValueRet, not_input_fees);
```

Comment:
Could collapse this to boolean fallback at this point.

Reply:
Done